runtime/src/main/java/org/eclipse/xpanse/runtime/XpanseApplication.java

```diff
@@ -21,6 +22,7 @@
 /**
  * Main entry class to xpanse runtime. This class can be directly executed to start the server.
  */
+@EnableCaching
```

Comment:
The annotation here is duplicated with the annotation above class `CaffeineCacheConfig`. Is it necessary?